core-bundle/assets/controllers/tooltip-controller.js

```diff
@@ -12,6 +12,14 @@ export default class extends Controller {
         if (this.observer) {
             this.observer.disconnect();
         }
+
+        Object.entries(this.selectors).forEach(([selector, options]) => {
```

Comment:
<img width="869" alt="" src="https://github.com/user-attachments/assets/f32bb450-faa0-4485-b64a-577e19761246">

Reply:
Fixed in 4aef7c2199